src/generate_docs.py

```diff
@@ -77,7 +77,10 @@ def main():
             current["metadata"][v] = schema
 
     for v, schema in directory_schemas.items():
-        if not v.isdigit() or int(v) < 2:
+        try:
+            a = float(v)
```

Comment:
If there is no use for 'a' down the code, I believe it could be just assert float(v).

Reply:
Also could it fail because v isn't a number? maybe another except will be needed

Reply:
Good catch(es)! Updated, let me know what you think. We don't seem to have cared about v not being a number before so I am just catching that ValueError in the same spot.